toolkit/cli/smart-contracts-commands/src/permissioned_candidates.rs

```diff
@@ -17,6 +18,8 @@ pub struct UpsertPermissionedCandidatesCmd {
 	payment_key_file: PaymentFilePath,
 	#[arg(long, short('c'))]
 	genesis_utxo: UtxoId,
+	#[arg(short, long, num_args = 1.., value_delimiter = ' ')]
+	governance_authority: Vec<MainchainKeyHash>,
```

Comment:
`docs` are not update and this parameter doesn't have a doc string as well, but I think this one should have it